Project/Assets/ML-Agents/Examples/FoodCollector/Scripts/FoodCollectorSettings.cs

```diff
@@ -44,5 +45,6 @@ void ClearObjects(GameObject[] objects)
     public void Update()
     {
         scoreText.text = $"Score: {totalScore}";
+        Academy.Instance.GetSideChannel<StatsSideChannel>().AddStat("TotalScore", totalScore);
```

Comment:
@vincentpierre Added to FoodCollector here. If we want to add to other examples, let's do so in a followup PR.

Reply:
Should we store the StatsSideChannel in a private property to avoid calling GetSideChannel every update?

Reply:
Yeah, probably - I'm not to worried about the performance, but we should set a good example...